beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -6617,13 +6618,14 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         &self,
         block_root: &Hash256,
     ) -> Result<Option<(LightClientBootstrap<T::EthSpec>, ForkName)>, Error> {
-        let Some((state_root, slot)) = self
-            .get_blinded_block(block_root)?
-            .map(|block| (block.state_root(), block.slot()))
-        else {
+        let runtime = tokio::runtime::Runtime::new().map_err(|_| Error::RuntimeShutdown)?;
```

Comment:
Could we reuse the `self.task_executor` to run a blocking task?

Reply:
yes! those changed in dde9274